packages/vscode-extension/package.json

```diff
@@ -307,8 +307,8 @@
           "when": "false"
         },
         {
-          "command": "fx-extension.cicdGuide",
-          "when": "false"
+          "command": "fx-extension.migrateV1Project",
```

Comment:
migrateV1Project should has been removed

Reply:
Thanks - let my sync my branch with upstream.

Reply:
Since there's no conflicts detected, I prefer to clean migrateV1Project automatically when merging.